app/views/categories/index.html.erb

```diff
@@ -1,17 +1,24 @@
 <h1 class="title">Categories</h1>
-<%= render 'form' %>
+<%= render 'category-form' %>
```

Comment:
For partial names, the convention I've seen for filenaming is that all words are separated by underscores rather than hyphens.